Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
So it looks like the `install_prerequisites.sh` scripts doesn't use the `-y` switch, causing unattended builds to fail in Vagrant.
```
==> default: After this operation, 21.9 MB of additional disk space will be used.
==> default: Do you want to continue?
==> default:  [Y/n]
==> default: Abort.
==> default: Reading package lists...
```

Adding the `-y` switch appears to fix that. Okay to add?